README.md

```diff
@@ -47,11 +47,11 @@ cargo --version
 ```
 
 Finally, the GUI has a few dependencies:
-- [`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) (On Debian/Ubuntu `apt install pkg-config`)
-- [`libxkbcommon`](https://xkbcommon.org/) for the dummy signer (On Debian/Ubuntu `apt install libxkbcommon-dev`)
-- `libelf` (On Debian/Ubuntu `apt install libelf-dev`)
-- [`fontconfig`](https://www.freedesktop.org/wiki/Software/fontconfig/) (On Debian/Ubuntu `apt install libfontconfig1-dev`)
-- [Vulkan Loader](https://github.com/KhronosGroup/Vulkan-Loader) (On Debian/Ubuntu `apt install libvulkan-dev`)
+- [`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) (On Debian/Ubuntu `sudo apt install pkg-config`)
+- [`libxkbcommon`](https://xkbcommon.org/) for the dummy signer (On Debian/Ubuntu `sudo apt install libxkbcommon-dev`)
+- `libelf` (On Debian/Ubuntu `sudo apt install libelf-dev`)
+- [`fontconfig`](https://www.freedesktop.org/wiki/Software/fontconfig/) (On Debian/Ubuntu `sudo apt install libfontconfig1-dev`)
+- [Vulkan Loader](https://github.com/KhronosGroup/Vulkan-Loader) (On Debian/Ubuntu `sudo apt install libvulkan-dev`)
```

Comment:
`sudo` is too specifc (not everyone has it), prefer a generalistic "run as root":

> (On Debian/Ubuntu `apt install <package>` as root)